Source/modules/webcl/WebCLException.cpp

```diff
@@ -215,9 +215,6 @@ void WebCLException::throwException(int& code, ExceptionState& es)
     case CL_INVALID_PROPERTY:
         es.throwWebCLException(WebCLException::INVALID_PROPERTY, WebCLException::invalidPropertyMessage);
         break;
-    case CL_INVALID_IMAGE_DESCRIPTOR:
```

Comment:
CL_INVALID_IMAGE_DESCRIPTOR is new in OpenCL 1.2. Not sure if it should be mapped to WebCLException::INVALID_IMAGE_**FORMAT**_DESCRIPTOR or a general failure.